autoPyTorch/optimizer/smbo.py

```diff
@@ -323,7 +350,8 @@ def run_smbo(self, func: typing.Optional[typing.Callable] = None
                                    n_jobs=self.n_jobs,
                                    initial_budget=initial_budget,
                                    max_budget=max_budget,
-                                   dask_client=self.dask_client)
+                                   dask_client=self.dask_client,
+                                   initial_configurations=self.initial_configurations)
```

Comment:
Also I just noticed that there is no unit testing for this. Do you think we should at least check that some initial configurations are valid for Australian?

Reply:
okay, I'll add a test.